<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>998</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>spxVoltMap</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="size32">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>729</width>
      <height>998</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(28, 33, 38)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGraphicsView" name="graphCanvas1">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>130</y>
       <width>681</width>
       <height>835</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(32,36,44);</string>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>42</red>
        <green>47</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foregroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </widget>
    <widget class="QComboBox" name="NominalVoltage">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>80</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>ALL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>765kV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>345kV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>154kV</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="RepresentativeVoltage">
     <property name="geometry">
      <rect>
       <x>288</x>
       <y>80</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>최대</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>최소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>평균</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="Unit">
     <property name="geometry">
      <rect>
       <x>439</x>
       <y>80</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>PU</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>kV</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>79</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color : rgb(255,255,255);
border : 0px solid red;</string>
     </property>
     <property name="text">
      <string>공칭전압 : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>79</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color : rgb(255,255,255);
border : 0px solid red;</string>
     </property>
     <property name="text">
      <string>대표전압 : </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>386</x>
       <y>79</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color : rgb(255,255,255);
border : 0px solid red;</string>
     </property>
     <property name="text">
      <string>단위 :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="redraw">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>79</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px solid white; color : white;</string>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphCanvas2">
     <property name="geometry">
      <rect>
       <x>424</x>
       <y>824</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(32,36,44);
</string>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>42</red>
        <green>47</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="foregroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </widget>
    <zorder>graphCanvas1</zorder>
    <zorder>label</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>redraw</zorder>
    <zorder>NominalVoltage</zorder>
    <zorder>RepresentativeVoltage</zorder>
    <zorder>Unit</zorder>
    <zorder>graphCanvas2</zorder>
   </widget>
   <widget class="QFrame" name="header_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>729</width>
      <height>80</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(28, 33, 38)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>0</y>
       <width>141</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color : white;</string>
     </property>
     <property name="text">
      <string>전압 분포도</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="title_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>28</y>
       <width>70</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255,255,255);
font-size : 12px;
background-color: rgb(84,54,95);
border-radius : 5px;</string>
     </property>
     <property name="text">
      <string>NA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="setting_btn">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px solid white; color : white;</string>
     </property>
     <property name="text">
      <string>설정</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
